BREAKING_CHANGES.md

```diff
@@ -43,3 +43,321 @@ This was just a wrapped component around `Flex`. You should use `Flex` instead.
+
+#### Envelop
+
+Replaced with `Mail`
+
+#### Equalizer
+
+Replaced with `SlidersHorizontal`
+
+#### ExclamationMarkCircle
+
+Replaced with `WarningCircle`
+
+#### Exit
+
+Replaced with `SignOut`
+
+#### Facebook
+
+Now exported from `@strapi/icons/symbols`
+
+#### FeatherSquare
+
+Now exported from `@strapi/icons/symbols`
+
+#### Gate
+
+Replaced with `CastleTurret`
+
+#### Github
+
+Now exported from `@strapi/icons/symbols`
+
+#### GlassesSquare
+
+Now exported from `@strapi/icons/symbols`
+
+#### Grid
+
+Replaced with `GridFour`
+
+#### Headphone
+
+Replaced with `Headphones`
+
+#### History
+
+Replaced with `ClockCounterClockwise`
+
+#### InformationSquare
+
+Now exported from `@strapi/icons/symbols`
+
+#### Json
+
+Replaced with `JsonField` and exported from `@strapi/icons/symbols`
+
+#### Landscape
+
+Replaced with `Images`
+
+#### LandscapeSmall
+
+Replaced with `Images`
+
+#### Layer
+
+Replaced with `ListPlus`
+
+#### LinkSmall
+
+Replaced with `Link`
+
+#### Media
+
+Replaced with `MediaField` and exported from `@strapi/icons/symbols`
+
+#### Medium
+
+Now exported from `@strapi/icons/symbols`
+
+#### MenuBurger
+
+Replaced with `List`
+
+#### MinusOutlined
+
+Replaced with `MinusCircle`
+
+#### Music
+
+Replaced with `MusicNotes`
+
+#### Number
+
+Replaced with `NumberField` and exported from `@strapi/icons/symbols`
+
+#### OnholdCarretDown
+
+Replaced with `CaretDown`
+
+#### OnholdCarretUp
+
+Replaced with `CaretUp`
+
+#### Paint
+
+Replaced with `PaintBrush`
+
+#### Password
+
+Replaced with `PasswordField`
+
+#### Picture
+
+Replaced with `Image`
+
+#### PicturePlus
+
+Replaced with `PlusCircle`
+
+#### PlaySquare
+
+Now exported from `@strapi/icons/symbols`
+
+#### Puzzle
+
+Replaced with `PuzzlePiece`
+
+#### Quote
+
+Replaced with `Quotes`
+
+#### QuoteClosed
+
+Replaced with `Quotes`
+
+#### Reddit
+
+Now exported from `@strapi/icons/symbols`
+
+#### Refresh
+
+Replaced with `ArrowClockwise`
+
+#### Relation
+
+Replaced with `RelationField` and exported from `@strapi/icons/symbols`
+
+#### Repeat
+
+Removed.
+
+#### RichText
+
+Replaced with `RichTextField` and exported from `@strapi/icons/symbols`
+
+#### Rotate
+
+Replaced with `ArrowsCounterClockwise`
+
+#### SearchIcon
+
+Removed, use `Search` instead.
+
+#### Seed
+
+Replaced with `Plant`
+
+#### Slideshow
+
+Replaced with `PresentationChart`
+
+#### Spark
+
+Replaced with `Sparkle`
+
+#### Spinner
+
+Replaced with `Loader`
+
+#### Strapi
+
+Now exported from `@strapi/icons/symbols`
+
+#### Text
+
+Replace with `TextField` and exported from `@strapi/icons/symbols`
+
+#### Twitter
+
+Replaced with `X` and exported from `@strapi/icons/symbols`
+
+#### Uid
+
+Replaced with `UidField` and exported from `@strapi/icons/symbols`
+
+#### Write
+
+Replaced with `Feather`
+
+## Icon has been removed
+
+The `Icon` component has been removed. It's primary function was to allow you to apply theme colors to icons, this is now doable with the actual icon:
```

Comment:
Doesn't this contradict this?

"We encourage users to use the `Icon` component to render icons as it gives access to the `theme` object in styled-components."